### PR TITLE
Disable scale unit feature

### PR DESCRIPTION
**Pull request by Omaaarz**
Added a capability to disable scale unit feature on scale units and remove triggers

### CODE REVIEW

File: src/ScaleUnitManagement/ScaleUnitFeatureManager/Common/RunDBSync.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public float Priority()
             return 3F;
         }
 
-        public void Run()
+        public void Run(bool isScaleUnitFeatureEnabled = true)
         {
             Console.WriteLine("Executing DbSync");
```
Comment (Antonstockmarr) [inline src/ScaleUnitManagement/ScaleUnitFeatureManager/Common/RunDBSync.cs:22]: Do we still want to print this if we are disabling scale unit feature?

**Review by Antonstockmarr — APPROVED** [review]

File: src/CLI/SyncDB.cs [hunk 2]
```diff
@@ -26,20 +26,21 @@ public static async Task Show(int input, string selectionHistory)
             await CLIMenu.ShowScreen(screen);
         }
 
-        private static async Task RunSyncDB(int input, string selectionHistory)
+        private static Task RunSyncDB(int input, string selectionHistory)
         {
-            ScaleUnitDBSync scaleUnitDBSync = new ScaleUnitDBSync();
```
Comment (AndreasHassing) [inline src/CLI/SyncDB.cs:31]: Is the `ScaleUnitDBSync` class no longer needed? If so, can probably remove in this PR.

File: src/ScaleUnitManagement/ScaleUnitFeatureManager/Common/SharedWebConfig.cs [hunk 3]
```diff
@@ -16,8 +16,8 @@ public static void Configure(WebConfig webConfig)
                     webConfig.UpdateXElementIfExists("AzureStorage.StorageConnectionString", scaleUnit.AzureStorageConnectionString);
             }
 
-            webConfig.AddKey("ScaleUnit.InstanceID", scaleUnit.ScaleUnitId);
-            webConfig.AddKey("ScaleUnit.Enabled", "true");
+            webConfig.AddKey("ScaleUnit.InstanceID", isScaleUnitFeatureEnabled ? scaleUnit.ScaleUnitId : "");
```
Comment (AndreasHassing) [inline src/ScaleUnitManagement/ScaleUnitFeatureManager/Common/SharedWebConfig.cs:19]: Does it matter what this value is if `ScaleUnit.Enabled == false`?

**Review by AndreasHassing — APPROVED** [review]
Minor comments 😊. Approving regardless.

File: src/ScaleUnitManagement/ScaleUnitFeatureManager/Common/SharedWebConfig.cs (hunk 3 above)
Comment (Omaaarz) [reply src/ScaleUnitManagement/ScaleUnitFeatureManager/Common/SharedWebConfig.cs:19]: It shouldn't but wanted to play it safe if there are any places where we miss checking the enablement of the feature before using the mnemonic.

File: src/CLI/SyncDB.cs (hunk 2 above)
Comment (Omaaarz) [reply src/CLI/SyncDB.cs:31]: The class is still being used as part of EnableScaleUnitFeature option. It's a wrapper that we don't need in our case here.

File: src/CLI/DisableScaleUnitFeature.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        private static Task RunDisableScaleUnitFeature(int input, string selectionHistory)
+        {
+            using (var context = ScaleUnitContext.CreateContext(sortedScaleUnits[input - 1].ScaleUnitId))
+            {
+                try
+                {
+                    new StopServices().Run();
+                    using (var webConfig = new WebConfig())
+                    {
+                        SharedWebConfig.Configure(webConfig, false);
```
[the remark this answers is not in the document]
Comment (Omaaarz) [reply src/CLI/DisableScaleUnitFeature.cs:39]: Makes sense.

**Pull request merged**